2018-Q2.md

```diff
@@ -0,0 +1,200 @@
+# Objectives & Key Results at source{d}
+
+## Q2 2018
+
+The list and tree below is automatically generated from the issues in this repository, using our [OKR GitHub tool](https://github.com/dennwc/okrs)
+
+-------------
+
+**Progress:** 0/2
+
+### Improve our Monthly Financials Reporting Process
+
+[Source page](https://github.com/src-d/okrs/issues/58)
+
+**Progress:** 0%
+
+* [P0] Create a Monthly Financials Dashboard
+* [P0] Refactor existing Excel spreadsheets
+* [P0] Successfully incorporate US expenses to Reporting Process
+
+### Vision: Being the leading company for ML on Code
+
+[Source page](https://github.com/src-d/okrs/issues/1)
+
+**Progress:** 0%
+
+* Store the world’s source code ([#5](https://github.com/src-d/okrs/issues/5))
+	* [P0] Store all public repositories from GitHub, BitBucket and self-hosted cgit ([#14](https://github.com/src-d/okrs/issues/14))
+		* [P0] Stable pipeline with rovers and borges
+		* [P0] Get computing and storage production-ready cluster to 60 machines
+	* [P1] Keep repositories never more than 30 days outdated ([#15](https://github.com/src-d/okrs/issues/15))
+		* [P1] Optimize borges to keep up with all repository updates within 30 days ([#27](https://github.com/src-d/okrs/issues/27))
+			* [P0] improve downloading of new repos by 50%
+			* [P1] improve update rooted repositories by 50%
+			* [P1] increase job throghtput by 30%
+			* [P2] decrease timeout errors by 30%
+			* [P2] decrease pod restarts by 50%
+* Being a better company to work at ([#4](https://github.com/src-d/okrs/issues/4))
+	* [P0] Successfully implement best practices for being a remote company ([#40](https://github.com/src-d/okrs/issues/40))
+		* [P0] Create a study on employees sentiment
+	* [P0] Having a more diverse hiring funnel (increase diversity in terms of gender and race)
+	* [P0] Improve instrumentation of the clusters for all teams ([#59](https://github.com/src-d/okrs/issues/59))
+		* [P0] Store pipeline cluster logs in a queryable manner ([#45](https://github.com/src-d/okrs/issues/45))
+			* [P0] Store logs
+			* [P1] Be able to query logs
+		* [P0] Support tracing logging
+	* [P1] Successfully execute OKRs methodology
+	* [P1] Remove technical friction between multiple teams ([#19](https://github.com/src-d/okrs/issues/19))
+		* [P0] Introduce monthly empathy sessions or other mechanism where different teams get to try other teams' tools
```

Comment:
Not only try, but also try to contribute something to the code, even if they're small things.